### PR TITLE
feat(collect-data-modal): apply changed collector state types and changed API spec

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
- selecting and applying necessary data from props
- apply changed API spec
- add translation

### CODE REVIEW

File: apps/web/src/services/asset-inventory/collector/shared/collector-data-modal/modules/CollectorDataDuplicationInner.vue [hunk 1]
```diff
@@ -138,24 +138,22 @@ const state = reactive({
...
+            if (status === JOB_STATE.SUCCESS) {
+                return 100 - state.succeededPercentage;
             }
-            return 0;
-        }),
-    },
+            return (state.jobTaskStatus.failed / state.jobTaskStatus.total) * 100;
+        }
+        return 0;
+    }),
```
Comment (sulmoJ) [inline apps/web/src/services/asset-inventory/collector/shared/collector-data-modal/modules/CollectorDataDuplicationInner.vue:156]: 🥳

**Review by sulmoJ — APPROVED** [review]
LGTM~~

File: apps/web/src/services/asset-inventory/collector/shared/collector-data-modal/modules/CollectorDataDuplicationInner.vue [hunk 2]
```diff
@@ -56,10 +56,10 @@
                 </div>
                 <div class="progress-bar">
                     <span class="succeeded-bar"
-                          :style="{ width: `${state.status.succeededPercentage}%` }"
+                          :style="{ width: `${state.succeededPercentage}%` }"
                     />
                     <span class="failed-bar"
-                          :style="{ width: `${state.status.failedPercentage}%` }"
+                          :style="{ width: `${state.failedPercentage}%` }"
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/shared/collector-data-modal/modules/CollectorDataDuplicationInner.vue:62]: praise: 🙌

**Review by yuda110 — APPROVED** [review]
L!G!T!M!

**Pull request merged**